lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -47,6 +58,36 @@ def release_languages(self, languages):
             enabled=True
         ).save()
 
+    def create_user(self):
+        """
+        Creates the user log in
+        """
+        # Create one user and save it to the database
+
+        self.user = UserFactory.build(username='test', email=self.email)
+        self.user.set_password(self.pwd)
+        self.user.save()
+
+        # Create a registration for the user
+        RegistrationFactory(user=self.user)
```

Comment:
Yes.  You should just remove it.  Perhaps it was needed in the past.  But these test no longer need it.